renovate.json

```diff
@@ -46,12 +46,12 @@
        "depNameTemplate": "kubernetes/kubernetes",
        "datasourceTemplate": "github-releases"
      },
-     {
-       "fileMatch": ["^lib/common.sh$"],
-       "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-       "depNameTemplate": "kubernetes/minikube",
-       "datasourceTemplate": "github-releases"
-     },
+    //  {
+    //    "fileMatch": ["^lib/common.sh$"],
+    //    "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
+    //    "depNameTemplate": "kubernetes/minikube",
+    //    "datasourceTemplate": "github-releases"
+    //  },
```

Comment:
Once we drop CAPM3 v1a4, we can uncomment this again and remove if/else condition from lib/common.sh